homeassistant/components/transmission/__init__.py

```diff
@@ -82,6 +90,16 @@ def refresh(event_time):
 
     track_time_interval(hass, refresh, scan_interval)
 
+    def add_torrent(service):
+        """Add new torrent to download."""
+        torrent = service.data[ATTR_TORRENT]
+        if torrent.startswith(('http:', 'https:', 'ftp:', 'magnet:')) or \
+           hass.config.is_allowed_path(torrent):
+            api.add_torrent(torrent)
+
```

Comment:
Maybe log a warning if we can't perform the operation?

Reply:
Yeah, I can add that!

Reply:
you don't need 'https:' if you use 'http' only

Reply:
I added both `http:` and `https:` for clarity, but it doesn't matter which solution to me. So I changed it.